Classes/Issues/Comments/Text/IssueCommentTextCell.swift

```diff
@@ -36,6 +37,17 @@ final class IssueCommentTextCell: DoubleTappableCell, ListBindable, CollapsibleC
         textView.reposition(width: contentView.bounds.width)
     }
 
+    // MARK: Accessibility
+
+    override var accessibilityLabel: String? {
+        get {
+            return contentView.subviews
```

Comment:
I am wondering if we should make this some extension on `UICollectionViewCell`? Something like `generatedAccessibilityLabel` or similar?

Reply:
Yes please!